www/src/Addons/Wii.tsx

```diff
@@ -43,91 +47,280 @@ export const wiiState = {
 	wiiExtensionSpeed: 400000,
 };
 
+const WII_EXTENSION_CONTROLS = [
```

Comment:
I've added ways for us to normalize formating, try running ´npm run format´
Some have reported it not really working, but worth trying!

Reply:
I'd have to go through this manually because it breaks up formatting in a way that isn't very easy on my eyes.